package.json

```diff
@@ -21,7 +21,7 @@
     "posttest": "npm run lint",
     "lint": "tslint --project ./tsconfig.json ./src/**/*.ts",
     "watch": "tsc -w",
-    "testonly": "mocha --reporter spec --full-trace ./dist/test/tests.js ./dist/test/asyncIteratorSubscription.js",
+    "testonly": "mocha --expose-gc --reporter spec --full-trace ./dist/test/tests.js ./dist/test/asyncIteratorSubscription.js",
```

Comment:
I think this needs to be `node --expose-gc ./node_modules/.bin/mocha --reporter spec --full-trace ./dist/test/tests.js ./dist/test/asyncIteratorSubscription.js` since the flag is needing to be passes to node not mocha.

Reply:
Confirmed! This is the change needed to make all the tests pass.

```
> node --expose-gc ./node_modules/.bin/mocha --reporter spec --full-trace ./dist/test/tests.js ./dist/test/asyncIteratorSubscription.js



  PubSub
    ✓ can subscribe and is called when events happen
    ✓ can unsubscribe

  AsyncIterator
    ✓ should expose valid asyncIterator for a specific event
    ✓ should trigger event on asyncIterator when published
    ✓ should not trigger event on asyncIterator when publishing other event
    ✓ register to multiple events
    ✓ should not trigger event on asyncIterator already returned
    ✓ should not register event listeners before next() is called

  GraphQL-JS asyncIterator
    ✓ should allow subscriptions
    ✓ should allow async filter
    ✓ should detect when the payload is done when filtering (506ms)
    ✓ should clear event handlers

  withFilter
    ✓ works properly with finite asyncIterators
    ✓ does not leak memory with promise chain #memory (3011ms)


  14 passing (4s)
```